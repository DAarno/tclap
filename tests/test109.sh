#!/bin/bash

TEST_RESULT=`dirname $0`/test109

../examples/test41 -h 2>&1 | head -n 5 | tee tmp.out

if cmp -s tmp.out $TEST_RESULT.out; then
    echo "OK"
    exit 0
else
    echo "FAIL"
    diff tmp.out $TEST_RESULT.out
    exit 1
fi
