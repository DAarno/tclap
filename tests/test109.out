
USAGE: 

   ../examples/test41  [-aDde] [-b <int>] [-m <str>] [--]
                       [--version] [-h] <int> <str> <int> ...
